github_pr.py

```diff
@@ -21,11 +22,12 @@ class OwnerCannotShipError(Exception):
 class NoMergeCommentError(Exception):
     pass
 
+logger = logging.getLogger()
 
 def check_required_fields(required, **args):
     for i in required:
         if args[i] is None:
-            print 'Error: %s parameter not set' % i
+            logging.error(' %s parameter not set', i.upper())
```

Comment:
I think you might be missing something like this: https://github.com/dataxu/jenkins-tools/blob/1369b1f0c3b7e9caf89dcfede446cc58668e61f8/scripts/blackout_dates/blackout_dates_calendar.py#L179